packages/ui-components/src/components/Inputs/HexcodeField.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { FormHelperText } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { TextField } from './TextField';
+
+const HexcodeFieldWrapper = styled.div`
+  margin-bottom: 1em;
+`;
+
+const HexcodeFieldContainer = styled.div`
+  display: flex;
+  align-items: flex-end;
+  margin-top: 1em;
+`;
+
+const HexcodePreview = styled.div`
+  background-color: ${({ backgroundColor }) => backgroundColor};
+  height: 100%;
+  width: 100%;
```

Comment:
```suggestion
  height: 100%;
  width: 100%;
```
Currently these are getting over written below. Are they meant to be max width?